### PR TITLE
Add `prepackage` script to regenerate all files

### PR DESCRIPTION
**Pull request by msujew**
Adds a new script that should to be run before publishing all npm packages. Regenerates all generated files in the workspace and compiles them.

### CODE REVIEW

**Comment by spoenemann** [comment]
Currently we run `npm publish` separately in the individual packages because we need to update versions as part of the process. Eventually we will document this. I think in this case the top-level prepackage script will not be considered, right?

File: examples/domainmodel/src/language-server/generated/module.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const languageMetaData: LanguageMetaData = {
 
 export const parserConfig: IParserConfig = {
     recoveryEnabled: true,
-    nodeLocationTracking: 'onlyOffset',
+    nodeLocationTracking: 'full',
```
Comment (dhuebner) [inline examples/domainmodel/src/language-server/generated/module.ts:17]: Q: Is this somehow related to the build?

**Comment by msujew** [comment]
Due to the comment by @spoenemann, I will close this for now.
